sig-list.md

```diff
@@ -51,7 +51,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
 
 | Name | Stakeholder SIGs |Organizers | Contact | Meetings |
 |------|------------------|-----------|---------|----------|
-|[Apply](wg-apply/README.md)|* API Machinery<br>* CLI<br>|* [Daniel Smith](https://github.com/lavalamp), Google<br>|* [Slack](https://kubernetes.slack.com/messages/wg-apply)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-wg-apply)|* Regular WG Meeting: [Tuesdays at 9:30 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
```

Comment:
I wonder if we should keep wg-apply information around? There will be references to it from various places, so it has historical value.

Reply:
Yeah I think we should leave wg apply stuff around, and edit it where it makes sense with links to the api expression stuff.